### PR TITLE
Modified the library, now people could send 10 variables at time

### PR DESCRIPTION
**Pull request by Metavix**
Changed format of name of functions
New example of new functions

### CODE REVIEW

File: Ubidots_GPRS.cpp [hunk 1]
```diff
@@ -7,11 +7,14 @@
 Ubidots::Ubidots(char* token){
 	_token = token;
 	_client.begin(BAUDRATE);
+	n = 10;
```
Comment (woakas) [inline Ubidots_GPRS.cpp]: change the name of this variable, example max_values

File: Ubidots_GPRS.cpp [hunk 2]
```diff
@@ -7,11 +7,14 @@
...
+	summ = 0;
```
Comment (woakas) [inline Ubidots_GPRS.cpp]: This variable must also rename, ex: lengthValues

**Comment by woakas** [comment]
@Metavix please change the name of the library files to Ubidots_Arduino_GPRS.(cpp, h) and the examples must be [CamelCase](https://en.wikipedia.org/wiki/CamelCase) ex: Ubidots_save_value.ino => UbidotsSaveValue.ino

**Comment by Metavix** [comment]
done! Mr woakas

**Pull request merged**